spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -472,3 +471,114 @@ definitions:
                     values:
                         - 0: Float RTK
                         - 1: Fixed RTK
+
+ - MSG_PVT:
+    id: 0x0208
+    short_desc: Position velocity time (UTC) solution
```

Comment:
See response below. I think the intension here was to make this very easy to consume, and having UTC is consistent with that.